src/Juvix/Compiler/Internal/Extra/Base.hs

```diff
@@ -158,24 +158,18 @@ subsHoles s = over leafExpressions helper
       ExpressionInstanceHole h -> fromMaybe e (s ^. at h)
       _ -> e
 
-instance HasExpressions FunctionClause where
-  leafExpressions f c = do
-    _clauseBody <- leafExpressions f (c ^. clauseBody)
-    _clausePatterns <- traverse (leafExpressions f) (c ^. clausePatterns)
-    pure FunctionClause {_clauseName = c ^. clauseName, ..}
-
 instance HasExpressions Example where
   leafExpressions f = traverseOf exampleExpression (leafExpressions f)
 
 instance HasExpressions FunctionDef where
   -- leafExpressions f (FunctionDef name ty clauses bi) = do
```

Comment:
Useless comment (but it was here before)

Reply:
I've removed it